credentials_manager/lib/credentials_manager.rb

```diff
@@ -1,3 +1,4 @@
+require 'fastlane_core'
```

Comment:
I'm not sure we can actually just pull in fastlane_core as a dependency to credentials_manager, as it'll create a circular dependency. When we publish updates and update dependencies they'd bump each other continuously. @KrauseFx can you weigh in on this, or am I talking nonsense?